src/org/usfirst/frc/team467/robot/RobotMap.java

```diff
@@ -71,13 +71,13 @@ public static void init(RobotID id) {
 	 */
 	public static final double NOMINAL_BATTERY_VOLTAGE = 12.0;
 
-	// The circumference of the wheels for use in determining distance in
+	// The circumference of the wheels for use in determining distance in inches in 
 	// position mode
-	public static final double WHEELPOD_CIRCUMFERENCE = 19.74; //20.5139; //21.43
+	public static final double WHEEL_CIRCUMFERENCE = 6*Math.PI; //20.5139; //21.43
 
 	// The number of encoder ticks per one revolution of the wheel. This is used
 	// for correctly determining RPM and position.
-	public static final int WHEELPOD_ENCODER_CODES_PER_REVOLUTION = 256;
+	public static final int WHEEL_ENCODER_CODES_PER_REVOLUTION = 1024;
```

Comment:
I don't see what the changes in this file had to do with acceleration limiting????

Reply:
I suppose they don't, but they are still changes we want so we can use this variable elsewhere.